kyaml/Makefile

```diff
@@ -35,6 +35,7 @@ vet:
 	go vet ./...
 
 openapi:
+	sed -i 's/"x-kubernetes-patch-merge-key": "containerPort"/"x-kubernetes-patch-merge-key": "name"/g' 'openapi/kubernetesapi/swagger.json'
```

Comment:
Above the make target please reference
#2767 and add a pointer to 
https://github.com/kubernetes/kubernetes/issues/39188
We should see how or if we can resolve this upstream.